src/scripts/modules/media/body/body.coffee

```diff
@@ -152,6 +152,31 @@ define (require) ->
             # NOTE: `.toggleClass()` explicitly requires a `false` (not falsy) 2nd argument
             $el.toggleClass('ui-has-child-title', $title.length > 0)
 
+          # Create subtitles in each chapter with corresponding ids as their href
+          total_subtitle = $temp.find("[data-depth='1']")
```

Comment:
ok. In that case can you make the CSS selector more specific? Just in case there are other elements that have a `data-depth='1'`.

Something like `...find("[data-type='section'][data-depth='1']")`